packages/nimble-components/src/utilities/tests/parameterized.ts

```diff
@@ -64,3 +30,117 @@ export const getSpecTypeByNamedList = <T extends { name: string }>(
     (x: T) => focusList.includes(x.name),
     (x: T) => disabledList.includes(x.name)
 );
+
+// eslint-disable-next-line no-restricted-globals
```

Comment:
Could be worth a comment both here and in the `.spec.ts` saying that clients should not disable this lint rule.